doc/Command-Reference.md

```diff
@@ -8987,6 +9002,82 @@ This command adds or deletes a member port to/from the already created portchann
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#portchannels)
 
+# Packet Trimming
+
+This section explains the various show commands and configuration commands available for users.
+
+### Packet Trimming Show commands
+
+This subsection explains how to display switch trimming configuration.
+
+**show switch-trimming global**
+
+This command displays switch trimming global configuration.
+
+- Usage:
+  ```bash
+  show switch-trimming global [OPTIONS]
+  ```
+
+- Options:
+  - _-j,--json_: display in JSON format
+
+- Example:
+  ```bash
+  admin@sonic:~$ show switch-trimming global
+  +-----------------------------+---------+
+  | Configuration               |   Value |
+  +=============================+=========+
+  | Packet trimming size        |     200 |
+  +-----------------------------+---------+
+  | Packet trimming DSCP value  |      20 |
+  +-----------------------------+---------+
+  | Packet trimming queue index |       2 |
+  +-----------------------------+---------+
+
+  admin@sonic:~$ show switch-trimming global --json
+  {
+      "size": "200",
+      "dscp_value": "20",
+      "queue_index": "2"
+  }
+  ```
+
+### Packet Trimming Config commands
+
+This subsection explains how to configure switch trimming.
+
+**config switch-trimming global**
+
+This command is used to manage switch trimming global configuration.
+
+- Usage:
+  ```bash
+  config switch-trimming global [OPTIONS]
```

Comment:
Do we need 'global' option here?

Reply:
@kperumalbfn yes. This is a generic style used in SONiC. I prefer to keep as is.